hw02/PlayerController.cs

```diff
@@ -20,12 +24,39 @@ 4. Jump Speed as a float
     represented as a component in Unity. How do we grab our player controller in code and where should we write this line of code in this
     C# document (PlayerController.cs)?
     */
+    public string playerName = "John";
+    public float movementSpeed = 5f;
+
+    public float gravityValue = 9.81f;
+
+    public float jumpSpeed = gravityValue * 2;
+
+    public PlayerController playerController;
+
+    public Vector3 moveInput;
+
+    public Rigidbody rb;
+   
     
     // Start is called before the first frame update
     void Start()
     {
+         Debug.Log("playerName" + playerName);
+         Debug.Log("movementSpeed" + movementSpeed);
+         Debug.Log("gravityValue" + gravityValue);
+         Debug.Log("jumpSpeed" + jumpSpeed);
+         playerController = GetComponent<PlayerController>();
```

Comment:
Close! If the script if not attached to the player object, we need to find the player object first.